examples/Makefile

```diff
@@ -35,34 +49,101 @@ opencv:
 	pachctl create pipeline -f opencv/montage.json
 	pachctl put file images@master -i opencv/images.txt
 	pachctl put file images@master -i opencv/images2.txt
-#	pachctl put file images@master -i opencv/images3.txt
+
+opencv-testing: 
+	pachctl create repo images
+	$(JQ) "$(STATS)" opencv/edges.json | pachctl create pipeline -f -
+	$(JQ) "$(STATS)" opencv/montage.json | pachctl create pipeline -f -
+	pachctl start transaction
+	pachctl start commit images@master
+	pachctl finish transaction
+	pachctl put file images@master -i opencv/images.txt
+	pachctl put file images@master -i opencv/images2.txt
+	pachctl put file images@master -i opencv/images3.txt
+	pachctl put file images@master -i opencv/images4.txt
+	pachctl delete file images@master:/w7RVTsv.jpg
+	pachctl finish commit images@master
+
+opencv-testing-delete:
+	-pachctl delete pipeline montage
+	-pachctl delete pipeline edges
+
+opencv-delete: opencv-testing-delete
+	-pachctl delete repo images
 
 ml/object-detection/frozen_inference_graph.pb:
 	wget -p http://download.tensorflow.org/models/object_detection/ssd_mobilenet_v1_coco_11_06_2017.tar.gz --directory-prefix /tmp
 	tar -C ml/object-detection --strip-components 1 -xvf /tmp/download.tensorflow.org/models/object_detection/ssd_mobilenet_v1_coco_11_06_2017.tar.gz ssd_mobilenet_v1_coco_11_06_2017/frozen_inference_graph.pb
 
-object-detection: ml/object-detection/frozen_inference_graph.pb
+
+object-detection-images-repo: ml/object-detection/frozen_inference_graph.pb
 	pachctl create repo images
+
+object-detection-no-images-repo: ml/object-detection/frozen_inference_graph.pb
+
+object-detection-demo: object-detection-no-images-repo
 	pachctl create repo training
```

Comment:
and maybe the same for each target - clean up before running.

Reply:
See above comment.